packages/sanes-chrome-extension/src/extension/background/createPersona/requestExtensionBadge.ts

```diff
@@ -0,0 +1,17 @@
+import { RequestHandler } from './requestHandler';
+
+/*global chrome*/
+export function updateExtensionBadge(): void {
```

Comment:
I think it would be nice if this just takes a `count: number` argument instead of letting `updateExtensionBadge` depend on `RequestHandler`.